libraries/AP_RangeFinder/RangeFinder.cpp

```diff
@@ -554,6 +562,13 @@ RangeFinder::RangeFinder_Status RangeFinder::status(uint8_t instance) const
     return state[instance].status;
 }
 
+void RangeFinder::handle_msg(mavlink_message_t *msg) {
```

Comment:
@magicrub There is a check here to only accept an input from a CC if the rangefinder type is a CC.  Assuming the user only has one CC rangefinder instance, this should negate the need for an instance id.  Or do we want to accept multiple CC rangefinder instances?  Or did I misunderstand your comment?